### PR TITLE
[SDKDevGuide] rc ab with requests

### PR DESCRIPTION
**Pull request by arifBurakDemiray**

### CODE REVIEW

File: sdk_dev_guide/current.md [hunk 1]
```diff
@@ -1966,258 +1966,244 @@ Countly.heatmap_whitelist = ["https://you.domain1.com", "https://you.domain2.com
...
+<pre>RCData CountlyInstance.<strong>getValue</strong>(key)</pre>
+<pre>CountlyInstance.<strong>registerDownloadCallback</strong>(callback)</pre>
+<pre>CountlyInstance.<strong>removeDownloadCallback</strong>(callback)</pre>
+<pre>CountlyInstance.<strong>clearAll</strong>()</pre>
+<h3 id="h_01JC39FGVSK8R4RHW2XC71GRWF">Implementation Details</h3>
 <p>
-  When it is given, it would trigger RC values to download (if enabled).
+  For config method <strong>enableRemoteConfigAutomaticTriggers</strong>
 </p>
+<pre>CountlyConfig<strong>.enableRemoteConfigAutomaticTriggers()<br><br>// Logic<br><br></strong>Enables automatic download of the remote config values (in a variable)<br>This will trigger remote config values to be downloaded in those situations:<br>- After a device id change, values first cache cleared then downloaded again<br>- After exiting from temporary device id mode, values are downloaded<br>- After enrolling into a variant, values are cache cleared and downloaded again<br>- After remote config consent is given after init, values are downloaded<br>- After init is finished if we are not in the temporary id mode, values are downloaded</pre>
```
Comment (turtledreams) [inline sdk_dev_guide/current.md:2001]: you wanted "cached/cleared" instead of "cache cleared"?

File: sdk_dev_guide/current.md [hunk 2]
```diff
@@ -1966,258 +1966,244 @@ Countly.heatmap_whitelist = ["https://you.domain1.com", "https://you.domain2.com
...
 <p>
-  <strong>Consent&nbsp;</strong>- Receiving "remote-config" consent should trigger
-  RC values to download. Removing that consent should clear the stored values.
+  For instance method <strong>downloadSpecificKeys</strong>
 </p>
+<pre>CountlyInstance.<strong>downloadSpecificKeys</strong>(keysToInclude, callback)<br><br><strong>// Valid values</strong><br>All nullable, only non null and not empty values are accepted<br>Non valid values are warned and ignored<br><br><strong>// Logic<br></strong>- If keysToInclude is not empty, a parameter will be added to include some keys to the remote <br>config fetch request. Server will return only specified values.<br>- If the keysToInclude is empty, this function will behave like downloading all values.<br><br>We only update the keys in the storage that are changed.</pre>
```
Comment (turtledreams) [inline sdk_dev_guide/current.md:2030]: it seems like we have to mention the types too, like string array and such

**Review by turtledreams — CHANGES_REQUESTED** [review]

**Comment by turtledreams** [comment]
good work

File: sdk_dev_guide/current.md (hunk 1 above)
Comment (arifBurakDemiray) [reply sdk_dev_guide/current.md:2001]: Hmm it would be more correct yes

File: sdk_dev_guide/current.md (hunk 2 above)
Comment (arifBurakDemiray) [reply sdk_dev_guide/current.md:2030]: oh I see

**Review by turtledreams — APPROVED** [review]

**Pull request merged**